mne/surface.py

```diff
@@ -1697,6 +1700,219 @@ def marching_cubes(image, level):
     return verts, triangles
 
 
+def _check_subject_dir(subject, subjects_dir):
+    """Check that the Freesurfer subject directory is as expected."""
+    subjects_dir = get_subjects_dir(subjects_dir, raise_error=True)
+    t1_fname = op.join(subjects_dir, subject, 'mri', 'T1.mgz')
+    brain_fname = op.join(subjects_dir, subject, 'mri', 'brain.mgz')
+    if not op.isfile(t1_fname) or not op.isfile(brain_fname):
+        raise ValueError('Freesurfer recon-all subject folder '
+                         'is incorrect or improperly formatted, '
+                         f'got {op.join(subjects_dir, subject)}')
+
+
+def _get_surface_RAS_volumes(base_image, subject_from, subjects_dir):
+    """Check if the image is in Freesurfer surface RAS space."""
+    import nibabel as nib
+    if not isinstance(base_image, nib.spatialimages.SpatialImage):
+        base_image = nib.load(base_image)
+    base_image = nib.MGHImage(
+        _get_img_fdata(base_image).astype(np.float32), base_image.affine)
+    fs_t1 = nib.load(op.join(subjects_dir, subject_from, 'mri', 'T1.mgz'))
+    if not np.allclose(base_image.affine, fs_t1.affine, atol=1e-6):
+        raise RuntimeError('The `base_image` is not aligned to Freesurfer '
+                           'surface RAS space. This space is required as '
+                           'it is the space where the anatomical '
+                           'segmentation and reconstructed surfaces are')
+    return base_image, fs_t1
+
+
+def _check_dig(dig):
```

Comment:
Good point, thanks

Reply:
Hmmm this is was a lot more complicated than I thought because the `dig` doesn't have the channel names...

I'll push what I have, I think it's better but it's still a bit convoluted I think.